hw1/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Jersey</title>
+</head>
+<body>
+    <div class="weather">
+        <div class="weather__top">
+            <div class="wt__item">
+                <img src="images/3.png">
+                <div class="wt__refresh"><img src="images/refresh.png"></div> 
+            </div>
+        </div>
+
+        <div class="weather__middle">
+          <div class="wm__item">
+            <div class="wm__degrees">79</div>
+            <div class="o">o</div>
```

Comment:
The class names should be meaningfull, to keep the code clean, .o .wt .wm should be replaced.